woocommerce/woocommerce-android
# include ssr into support tickets

Closes: #7912 


### Description


Written w/ @malinajirka 

**Context:** p8wKgj-3cY-p2

This PR intends to fetch the SSR from the selected site (provided the user is logged in) and adds a new field to support tickets displaying the SSR result. 

**Example:** 5730923-zd-woothemes

Notes:
We decided that for the first iteration we will not wait to be sure the response comes in 100% of the time. We will fetch it when the user accesses the Help screen (provided they are logged in) and if the server responds we pass it to the ticket if the server doesn't we pass null and no SSR is added. For future interactions, we will evaluate the frequency of each response and decide if we should wait for the response, add a loading icon, etc or leave it as is.

We had to fix the existing tests for the HelpViewModel and will add tests for the added code soon. 


### Testing instructions


While logged into your account, go to the settings menu - Help & Support - Contact support (make sure the email you are using for contact is not an Automattic email) and send a support ticket. 

Then, head to Zendesk, search for your ticket and make sure there's a new field called System Status Report (right below WP Mobile App Other Info) and make sure the SSR is present and formatted properly. 

### Images/gif

<img width="257" alt="Screen Shot 2022-11-24 at 4 52 56 PM" src="https://user-images.githubusercontent.com/30724184/203825015-b6bdc49b-1461-4759-ab14-9677aebd2f33.png">
